docs/SOURCES.md

```diff
@@ -1,255 +1,972 @@
+| 🇸🇯 Svalbard & Jan Mayen | [MET Norway](#met-norway)                          | ~10 days |
+| 🇸🇪 Sweden               | [SMHI](#smhi)                                      | 15 days  |
+| 🇹🇼 Taiwan               | [CWA](#central-weather-administration) 🔐          | 7 days   |
+| 🇹🇷 Türkiye              | [MGM](#meteoroloji-genel-müdürlüğü)                | 5 days   |
+| 🇬🇧 United Kingdom       | [Met Office](#met-office) 🔐                       | 7 days   |
+| 🇺🇸 United States        | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇮 U.S. Virgin Is.      | [NWS](#national-weather-service)                   | 7 days   |
+| 🇻🇦 Vatican City         | [Meteo AM](#servizio-meteo-am)                     | 5 days   |
+
+## Worldwide sources
+
+### Open-Meteo
+**[Open-Meteo](https://open-meteo.com/)** is a weather data provider based in Bürglen, Switzerland. It is the only free and open source weather source on this list, and probably also the most privacy-friendly. When we don’t support your national weather source, we suggest using **Open-Meteo** as your primary weather source.
+
+| Feature                        | Detail                                                                           |
+|--------------------------------|----------------------------------------------------------------------------------|
+| 🔐 **API key**                 | Not required                                                                     |
+| 🗺️ **Coverage**               | 🌐 Worldwide (some features may not be available for all locations)              |
+| ▶️ **Current observation**     | Available: most recent real-time data from the nearest weather station           |
+| 📆 **Daily forecast**          | Up to 15 days                                                                    |
+| ⏱️ **Hourly forecast**         | Up to 16 days                                                                    |
+| 😶‍🌫️ **Air quality**         | Available                                                                        |
+| 🤧 **Pollen**                  | Available in Europe for: Alder, Birch, Grass, Mugwort, Olive, and Ragweed        |
+| ☔ **Precipitation nowcasting** | Available (works best in Europe at the moment)                                   |
+| ⚠️ **Alerts**                  | Not available: consider adding [Accuweather](#accuweather) as a secondary source |
+| 📊 **Normals**                 | Not available: consider adding [Accuweather](#accuweather) as a secondary source |
+| 🧭 **Reverse geocoding**       | Not available: will not show the name of device location                         |
+
+For the United States, [Forecast Advisor](https://www.forecastadvisor.com/) has temperature and precipitation 1-3 days accuracy comparison by city between the following sources: [Accuweather](#accuweather), [NWS](#national-weather-service), **Open-Meteo** and [Pirate Weather](#pirate-weather).
```

Comment:
Typo (to replace everywhere): Accuweather -> AccuWeather

Reply:
Will fix.